terraform/rabbitmq/rabbitmq.conf

```diff
@@ -17,4 +17,19 @@ vm_memory_high_watermark.relative = 0.75
 
 ## Defines the threshold of memory using at which publishers to the queue are throttled 
 ## https://www.rabbitmq.com/disk-alarms.html
-disk_free_limit.relative = 0.75
\ No newline at end of file
+disk_free_limit.relative = 0.75
+
+## Defines the SSL configuration for RabbitMQ
+## https://www.rabbitmq.com/docs/ssl.html
+listeners.ssl.default = 5671
+ssl_options.cacertfile = /rabbitmq/certs/ca.pem
+ssl_options.certfile = /rabbitmq/certs/rabbit.crt
+ssl_options.keyfile = /rabbitmq/certs/rabbit.key
+ssl_options.verify = verify_peer
+ssl_options.fail_if_no_peer_cert = false
+management.ssl.port = 15671
+management.ssl.cacertfile = /rabbitmq/certs/ca.pem
+management.ssl.certfile = /rabbitmq/certs/rabbit.crt
+management.ssl.keyfile = /rabbitmq/certs/rabbit.key
+management.ssl.verify = verify_none
```

Comment:
this conf is mounted on windows too. Is it taken into account ? If this is not the case, do not upload this conf file on windows